### PR TITLE
LV_LIST: Hide icon/image creation behind #if LV_USE_IMG 

### PR DESCRIPTION
**Pull request by FlashForger**
In case images are not used in projects but lists are

### CODE REVIEW

**Comment by FlashForger** [comment]
I don't understand what is the problem

File: src/extra/widgets/list/lv_list.c [hunk 1]
```diff
@@ -81,11 +81,13 @@ lv_obj_t * lv_list_add_btn(lv_obj_t * list, const char * icon, const char * txt)
...
 
+#if LV_USE_IMG == 1
     if(icon) {
         lv_obj_t * img = lv_img_create(obj);
         lv_img_set_src(img, icon);
     }
-
+#endif
+    
     if(txt) {
```
Comment (kisvegabor) [inline src/extra/widgets/list/lv_list.c:91]: What about adding `#if LV_USE_LABEL` here?

**Review by kisvegabor — COMMENTED** [review]
Thanks! 
Following your idea, I had a suggestion.

File: src/extra/widgets/list/lv_list.c (hunk 1 above)
Comment (FlashForger) [reply src/extra/widgets/list/lv_list.c:91]: I can hardly imagine a scenario where someone would use Lists without Labels (I can hardly imagine anyone using this library without labels). It'll mean only using lists with buttons, that also have no labels, i.e. empty buttons or with images. But technically possible so it's your call.

Comment (kisvegabor) [reply src/extra/widgets/list/lv_list.c:91]: Makes sense :slightly_smiling_face: 

The CI complains about some code formatting issues. Please run `scripts/code-format.py` (astyle needs to be installed)

Comment (FlashForger) [reply src/extra/widgets/list/lv_list.c:91]: I ran it, but I don't understand, what should I correct.

Comment (embeddedt) [reply src/extra/widgets/list/lv_list.c:91]: Once you run it, it should have reformatted your source code. Now you just have to run `git commit` and push it to your fork.

(If you'd rather just have me correct the formatting, I can also do that.)

Comment (FlashForger) [reply src/extra/widgets/list/lv_list.c:91]: Ok, this time it worked

**Comment by embeddedt** [comment]
Thanks!

**Pull request merged**